### PR TITLE
Freezing deps for 1.8 release

### PR DESCRIPTION
**Pull request by Gmerold**
# Description

- Freezes deps for 1.8 release
- Updates CI workflows

# Checklist:

- [x] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that validate the behaviour of the software
- [x] I validated that new and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/workflows/agw.pr_main.yml [hunk 1]
```diff
@@ -1,17 +1,21 @@
-name: main-branch-pull-request-created
+name: 1.6-branch-pull-request-created
 on:
   pull_request:
     branches:
-      - main
+      - 1.6
```
Comment (saltiyazan) [inline .github/workflows/agw.pr_main.yml:5]: you probably wanted to write `1.8` :)

File: .github/workflows/agw.pr_main.yml [hunk 2]
```diff
@@ -1,17 +1,21 @@
-name: main-branch-pull-request-created
+name: 1.6-branch-pull-request-created
```
Comment (saltiyazan) [inline .github/workflows/agw.pr_main.yml:1]: you probably wanted to write 1.8 :)

File: .github/workflows/agw.pr_main.yml (hunk 1 above)
Comment (Gmerold) [reply .github/workflows/agw.pr_main.yml]: Yeah :D

File: .github/workflows/agw.pr_main.yml (hunk 2 above)
Comment (Gmerold) [reply .github/workflows/agw.pr_main.yml]: Fixed.

**Review by saltiyazan — APPROVED** [review]

**Review by ghislainbourgeois — APPROVED** [review]

**Review by gruyaume — APPROVED** [review]

**Pull request merged**